xml/deployment_prep_zseries_prep_secure_boot.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<!--
+   fs 02/28/2006:
+     Do not change the following words/phrases (except in URLs,
+     packagenames etc.)in all zSeries documents, because they are trademarks:
+      - Redbook, Redpiece, Redpaper
+      - developerWorks
+      - ESCON
+      - HiperSockets
+-->
+<sect1 version="5.0" xml:id="sec-zseries-prep-secure-boot"
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>Secure boot</title>
+
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker/>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+
+ <para>
+  For the secure boot functionality to work on a &zseries; system, the following conditions must be met.
+ </para>
+ <itemizedlist>
+ <listitem>
+ <para>
+ The machine must be z15 T01, z15 T02, LinuxONE III LT1, LinuxONE III LT2, or later model.
+ </para>
+ </listitem>
+ <listitem>
+ <para>
+ You must use LPAR (secure boot is not supported on z/VM and &kvm;).
+ </para>
+ </listitem>
+ <listitem>
+ <para>
+ The LPAR must have secure boot enabled.
+ </para>
+ </listitem>
+ <listitem>
+ <para>
+ You must use SCSI (FCP) disks (secure boot is not supported on DASD).
+ </para>
+ </listitem>
+ </itemizedlist>
+ <note>
+ <title>Hardware migration</title>
+ <para>
+ In case you migrate to a different machine (for example, from z13 to z15), make sure that the LPAR on the target machine must have the secure boot state of the system on its disk.
```

Comment:
```suggestion
 In case you migrate to a different machine (for example, from z13 to z15), ensure
 that the LPAR on the target machine has the secure boot state of the system on its disk.
```

Reply:
I have wrapped this very long line. I think you are still having issues with VSCode not line-wrapping at 79 characters...